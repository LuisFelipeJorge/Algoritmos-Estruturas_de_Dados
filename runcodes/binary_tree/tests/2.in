8
0 1 2
1 5 6
2 3 4
3 -1 -1
4 -1 -1
5 7 -1
6 -1 -1
7 -1 -1