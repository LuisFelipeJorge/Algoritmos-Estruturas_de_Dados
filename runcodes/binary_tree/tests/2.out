no 0: pai = -1, altura = 4, grau = 2, filhos = (1,2), tipo = raiz
no 1: pai = 0, altura = 3, grau = 2, filhos = (5,6), tipo = interno
no 5: pai = 1, altura = 2, grau = 1, filhos = (7,-1), tipo = interno
no 7: pai = 5, altura = 1, grau = 0, filhos = (-1,-1), tipo = folha
no 6: pai = 1, altura = 1, grau = 0, filhos = (-1,-1), tipo = folha
no 2: pai = 0, altura = 2, grau = 2, filhos = (3,4), tipo = interno
no 3: pai = 2, altura = 1, grau = 0, filhos = (-1,-1), tipo = folha
no 4: pai = 2, altura = 1, grau = 0, filhos = (-1,-1), tipo = folha
